source/app/service-providers/assets/get-config-template.ts

```diff
@@ -211,8 +211,8 @@ export default function getConfigTemplate (): ConfigOptions {
     },
     system: {
       deleteOnFail: false, // Whether to delete files if trashing them fails
-      leaveAppRunning: false, // Whether to leave app running in the notification area (tray)
-      avoidNewTabs: true // Whether to avoid opening new tabs for documents if possible
+      avoidNewTabs: true, // Whether to avoid opening new tabs for documents if possible
+      leaveAppRunning: false
```

Comment:
This change removes a comment and swaps the order. An error when merging? I suggest don't change `get-config-template.ts`.